addons/web_editor/static/src/scss/web_editor.common.scss

```
@@ -276,6 +288,12 @@ img.ms-auto, img.mx-auto {
 .base-fs {
     @include font-size($font-size-base);
 }
+// Equivalent "font-size" only for the Odoo own "o_small" class. Note that the
+// "o_small" class currently also changes the font-size only but this is the
```

```suggestion
// "o_small" class currently also changes the font-size only but this is to
```
But not a big deal, feel free to change if it really annoys you and I'll r+ again

---

@rdeodoo You can r+ again https://github.com/odoo/odoo/compare/a4d8b3dd5a10bce4ab5113eb11e7cda0cef122b7..bfb0da3cb238a25450ea475b449627c66fbb3c61#diff-a92fcce4c7f53cfad14d3877bd8777c969fa7859ab32a0ffdfb4de73dc7a8013R292

(I rebased at the same time by mistake apparently)